third_party/lua-yaml/lyaml.cc

```diff
@@ -106,7 +106,7 @@ struct lua_yaml_dumper {
  * the block scalar style. Setting this flag, makes the encoder use the block
  * scalar style for all multiline strings.
  */
-static bool yaml_pretty_multiline;
+static bool yaml_pretty_multiline = true;
```

Comment:
The comment above states the old behavior as default and the comment should be updated.

Reply:
Fixed:
```C
  /**
   * By default, all strings that contain '\n' are encoded in the block scalar
   * style. Setting this flag to false, makes the encoder use default yaml style
   * with excessive newlines for all strins without "\n\n" substring. This is a
   * compatibility-only feature.
   */
```

Reply:
Typo: strins.